pkg/proxy/winuserspace/proxier.go

```diff
@@ -39,13 +39,17 @@ import (
 
 const allAvailableInterfaces string = ""
 
+var localhostIPv4 = net.ParseIP("127.0.0.1")
+var localhostIPv6 = net.ParseIP("::1")
+
 type portal struct {
 	ip         string
 	port       int
 	isExternal bool
 }
 
-type serviceInfo struct {
+// ServiceInfo contains information and state for a particular proxied service
+type ServiceInfo struct {
```

Comment:
same question here, what's the reason for exporting this?

Reply:
iirc the other `ServiceInfo` instances were doing this, but i need to double check... will get back on this one

Reply:
ah yes found it here https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/userspace/proxier.go
 ...  Im assuming here that the Linux kube-proxy is the gold standard... and this is a deviation from that:

```
// ServiceInfo contains information and state for a particular proxied service
54 | type ServiceInfo struct {
55 |  
```

is what we do in pkg/proxy/userspace/proxier.go. Somehow that comment, and the struct exporting, both changed when copied to windows .... its a minor difference but, when comparing the two implementations...  it makes you wonder why one is not exported, while the other **is** exported.